### PR TITLE
[risk=no][RW-7234] Set RAS module timestamp in DB to null for RAS test user

### PR DESCRIPTION
**Pull request by aweng98**
Modify `project.rb set-access-module-timestamps` cmd to accommodate RAS test user.

New cmd e.g.
`./project.rb set-access-module-timestamps --profile-user "puppet-access-tester-1@fake-research-aou.org" --ras-user "puppeteer-tester-3@fake-research-aou.org"`

Cmd modifies following:
- For `profile-user`, reset PROFILE module `completion_time` timestamp to a fixed value.
- For `ras-user`, set RAS module `completion_time` timestamp to null.

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: api/tools/src/main/java/org/pmiops/workbench/tools/SetAccessModuleTimestamps.java [hunk 1]
```diff
@@ -44,58 +41,71 @@ public class SetAccessModuleTimestamps {
...
 
   @Bean
   public CommandLineRunner run(AccessModuleService accessModuleService, UserDao userDao) {
     return (args) -> {
       final CommandLine opts = new DefaultParser().parse(OPTIONS, args);
-      final String username = opts.getOptionValue(USER_OPT.getLongOpt());
-      dbUser = userDao.findUserByUsername(username);
-      if (dbUser == null) {
-        throw new IllegalArgumentException(String.format("User %s not found!", username));
+      final String profileUsername = opts.getOptionValue(PROFILE_USER_OPT.getLongOpt());
```
Comment (evrii) [inline api/tools/src/main/java/org/pmiops/workbench/tools/SetAccessModuleTimestamps.java:87]: Out of curiosity, what does the getLongOpt method do?

**Review by evrii — APPROVED** [review]
Looks good to me.

File: api/tools/src/main/java/org/pmiops/workbench/tools/SetAccessModuleTimestamps.java (hunk 1 above)
Comment (aweng98) [reply api/tools/src/main/java/org/pmiops/workbench/tools/SetAccessModuleTimestamps.java:87]: Returns long name of the option. https://www.demo2s.com/java/apache-commons-option-getlongopt.html

**Review by yonghaoy — APPROVED** [review]

File: api/tools/src/main/java/org/pmiops/workbench/tools/SetAccessModuleTimestamps.java [hunk 2]
```diff
@@ -44,56 +41,69 @@ public class SetAccessModuleTimestamps {
...
+          .build();
+
+  private static final Options OPTIONS =
+      new Options().addOption(PROFILE_USER_OPT).addOption(RAS_USER_OPT);
 
   private static final Timestamp PROFILE_CONFIRMATION_TIMESTAMP =
       Timestamp.from(Instant.parse("2011-08-01T00:00:00.00Z"));
 
-  private static DbUser dbUser;
+  private static DbUser dbProfileUser;
```
Comment (jmthibault79) [inline api/tools/src/main/java/org/pmiops/workbench/tools/SetAccessModuleTimestamps.java:66]: nit: these can be local variables inside the run() method

**Pull request merged**